scripts/install_boost.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -ex
+
+BOOST_VERSION="$1"
+BOOST_MAJOR_MINOR=$(echo "${BOOST_VERSION}" | cut -d. -f-2)
+BOOST_MAJOR=$(echo "${BOOST_VERSION}" | cut -d. -f-1)
+BOOST_MINOR=$(echo "${BOOST_MAJOR_MINOR}" | cut -d. -f2-)
+BOOST_PATCH=$(echo "${BOOST_VERSION}" | cut -d. -f3-)
+BOOST_VERSION_U="${BOOST_MAJOR}_${BOOST_MINOR}_${BOOST_PATCH}"
+
+echo "using python : : /usr/bin/python${PYTHON_VERSION} ;\n" \
+     "using gcc : : ${CXX} ;\n" \
+     > ${HOME}/user-config.jam
+
+mkdir _boost
+cd _boost
+
+wget -q https://sourceforge.net/projects/boost/files/boost/${BOOST_VERSION}/boost_${BOOST_VERSION_U}.tar.gz
+tar -xzf boost_${BOOST_VERSION_U}.tar.gz
+
+cd boost_${BOOST_VERSION_U}
+sh bootstrap.sh
+./b2 install -j4 variant=release toolset=gcc \
+    --with-system \
+    --with-regex \
+    --with-filesystem \
+    --with-thread \
+    --with-python \
```

Comment:
We're missing boost::iostreams for VDB, which means these extra library components:
```
    --with-atomic \
    --with-chrono \
    --with-date_time \
    --with-iostreams \
```

Reply:
I just pushed a new image with added boost libraries if you want to try?

Reply:
Thanks, this part is working now!